From: Deepak K <deepak.k@ti.com>
Date: Thu, 28 Apr 2011 19:50:23 +0000 (-0500)
Subject: OMAP4 : UART : Removes recalib baud rate from DPLL Framework
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=533f4d2b83d69e3937ef21e18482836c18e160b5

OMAP4 : UART : Removes recalib baud rate from DPLL Framework

This removes the recalibrate baud rates for UART from the DPLL
framework. Instead it would be registered as call back to the
framework.

Change-Id: I035c1d298c995801d4dce15e698310d578665de5
Signed-off-by: Deepak K <deepak.k@ti.com>
---

Index: kernel/arch/arm/mach-omap2/dpll-44xx.c
===================================================================
--- kernel.orig/arch/arm/mach-omap2/dpll-44xx.c	2011-05-21 00:43:03.270847085 +0900
+++ kernel/arch/arm/mach-omap2/dpll-44xx.c	2011-05-21 00:43:24.420846790 +0900
@@ -782,9 +782,6 @@
 	} else
 		pr_debug("%s: DPLL_PER entered Low Power bypass\n",__func__);
 
-	/* recalibarte UART rate for half rates */
-	omap_uart_recalibrate_baud(1);
-
 	__raw_writel(1, OMAP4430_CM_L4_WKUP_CLKSEL);
 
 	/* never de-assert CLKREQ while in DPLL cascading scheme */
@@ -967,9 +964,6 @@
 		pr_err("%s: failed to restore DPLL_PER bypass clock\n",
 				__func__);
 
-	/* recalibarte UART rate for normal clock rates */
-	omap_uart_recalibrate_baud(0);
-
 	/* restore CORE clock rates */
 	ret = clk_set_rate(div_core_ck, (div_core_ck->parent->rate /
 				(1 << state.div_core_ck_div)));
